### PR TITLE
chg: Keep connection alive between requests

### PR DESCRIPTION
**Pull request by JakubOnderka**
Keep connection alive between requests, so PyMISP can reuse existing connection (no need to establish new encrypted channel, bigger TCP window etc.), so sending multiple requests should be much faster.

### CODE REVIEW

**Comment by Rafiot** [comment]
Looks great, thank you!

**Comment by Rafiot** [comment]
I just saw that it was a draft. @JakubOnderka should I merge it? Or do you want to test it further?

File: pymisp/api.py [hunk 1]
```diff
@@ -114,6 +114,7 @@ def __init__(self, url: str, key: str, ssl: bool = True, debug: bool = False, pr
         self.auth: Optional[AuthBase] = auth
         self.tool: str = tool
         self.timeout: Optional[Union[float, Tuple[float, float]]] = timeout
+        self.__session = requests.Session()  # use one session to keep connection between requests
```
Comment (chrisinmtown) [inline pymisp/api.py:117]: will this be configurable?

**Comment by chrisinmtown** [comment]
This seems like a *major* change, but there's no option to enable/disable it, and there's absolutely no documentation of the new feature. Is this usually how PyMISP works?

**Comment by Rafiot** [comment]
It is a simple optimization, and it won't change anything for the user, except that it will be faster for code running multiple calls to MISP.

**Comment by JakubOnderka** [comment]
> I just saw that it was a draft. @JakubOnderka should I merge it? Or do you want to test it further?

Yes, it can be merged. I just tested it and for 100 requests, the speed up is big (from 15 seconds to 5 seconds for one attribute restSearch request).

**Pull request merged**